tests/functional/FieldTest.php

```diff
@@ -59,6 +59,14 @@ public function testFieldCreate()
         $this->assertStringContainsString('Success', $this->getErrorOutputRaw());
         $this->drush('field:create', ['unish_article', 'beta'], ['existing-field-name' => 'field_test3', 'field-label' => 'Body', 'field-widget' => 'text_textarea_with_summary'], null, null, self::EXIT_ERROR);
         $this->assertStringContainsString('Field with name \'field_test3\' already exists on bundle \'beta\'', $this->getErrorOutputRaw());
+
```

Comment:
```suggestion
if (version_compare(\Drupal::VERSION, '10.1.0') < 0) {
    $this->markTestSkipped('Allowed formats available since Drupal 10.1.0');
}
```

Reply:
Okay, cool. Will these tests currently ever run by doing this? Is Drush being tested using multiple Drupal versions, including Drupal 10.1?